ci/requirements-docs-3.6.yml

```diff
@@ -22,6 +22,7 @@ dependencies:
   - plumbum
   - psycopg2
   - pyarrow>=0.6.0
+  - pymapd
```

Comment:
does this require a version constraint?

Reply:
you're right! it is better to pin the version here. thanks!